platform/security/servlet/security-servlet-logout/src/test/java/org/codice/ddf/security/servlet/logout/LogoutServletTest.java

```diff
@@ -134,52 +128,7 @@ public void testLocalLogoutNotBasicOrPki() {
       fail(e.getMessage());
     }
     verify(httpSession).invalidate();
-    verify(printWriter).write("{ \"mustCloseBrowser\": false, \"redirectUrl\": \"\" }");
-  }
-
-  @Test
-  public void testLocalLogoutredirectUrlNotBasicOrPki() {
-    // Used for detecting basic auth
-    localLogoutServlet.setRedirectUrl("redirectUrlAddress.com");
-    when(request.getHeaders(anyString()))
-        .thenReturn(Collections.enumeration(Collections.emptyList()));
-
-    // used for detecting pki
-    when(request.getAttribute("javax.servlet.request.X509Certificate")).thenReturn(null);
-
-    SecurityTokenHolder securityTokenHolder = mock(SecurityTokenHolder.class);
-    when(httpSession.getAttribute(SecurityConstants.SECURITY_TOKEN_KEY))
-        .thenReturn(securityTokenHolder);
-    try {
-      localLogoutServlet.doGet(request, response);
-    } catch (ServletException e) {
-      fail(e.getMessage());
-    }
-    verify(httpSession).invalidate();
-    verify(printWriter)
```

Comment:
it looks like this test was actually validating a set "redirect url". is there any way we can add a test to validate the actual logout url set by the config? looks like most the tests here now are non happy path tests, or they just check the `mustCloseBrowser` bool but not the url.

Reply:
This ticket was originally done last year by modifying the LogoutServlet module to set a redirect URL. That fix did not work in some downstream projects so this PR is reimplementing the logout redirect in a different way and undoing those previous changes in this module. I added the tests you mentioned when I worked on the previous fix for this but once this is merged, the "redirect URL" will no longer exist and the LogoutServlet module will not be involved in the signout button redirect